0xProject/0x-monorepo
# Contracts monorepo II

## Description
Refactors `@0x/contracts-core` into three packages:
* `@0x/contracts-core` All contracts
* `@0x/contracts-test-utils` TS contract testing utils
* `@0x/contracts-multisig` Mutisig contracts

`@0x/contracts-core` has a dependency on `@0x/contracts-multisig` and imports those contracts using NPMResolver.

There were a couple of bugs in resolver/compiler, but they're fixed now.

Publishing tests fail because of that [PR](https://github.com/0xProject/0x-monorepo/pull/1312)

## Testing instructions

Compile contracts and see that multisig contracts were imported from an NPM package. (In artifacts)
Run multisig tests separate from core tests.

## Types of changes
* New feature (non-breaking change which adds functionality)
* Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
